### ISSUE TITLE
`WooCommerce.Commenting.CommentHooks.MissingSinceComment` should not throw an error if the hook is documented elsewhere in the codebase

### ISSUE THREAD

**Issue by coreymckrill** [woocommerce/woocommerce-sniffs#37]
In cases where a hook is fired in multiple different places in the codebase, it is a common convention to add the full doc block (including `@since`) in _one_ of the places, and then in the other places just add a comment that looks something like this:

```php
/** This filter is documented in includes/class-wc-logger.php. */
```

However, it does not appear that phpcs accounts for this, and is insisting that I add a `@since` tag to the above example.

**Comment by kallookoo** [comment]
Any progress to solve it?
